README.md

```diff
@@ -134,7 +134,7 @@ An example minimal configuration would look like this, within your robot’s "se
   "type": "mlmodel",
   "attributes": {
     "model_name": "efficientdet-lite4-detection",
-    "model_repository_path": "/path/to/.viam/triton/repository"
+    "model_path": "${packages.ml_model.FaceDetector}"
```

Comment:
I'm not convinced we should be removing `model_repository_path`, as that precludes using the triton server against a model repository positioned on the device already, which was the existing mechanism for using the triton module.

Instead, the server should continue to honor `model_repository_path` if provided. Otherwise, expect a `model_path` and do the symlinking.